adcctestdata/dump_pyscf.py

```diff
@@ -217,5 +217,17 @@ def dump_pyscf(scfres, out):
     mmp.create_dataset("elec_1",
                        data=scfres.mol.intor_symmetric("int1e_r", comp=3))
 
+    magm = data.create_group("magnetic_moments")
+    derivs = data.create_group("derivatives")
+    with scfres.mol.with_common_orig([0.0, 0.0, 0.0]):
+        magm.create_dataset(
+            "mag_1",
+            data=0.5 * scfres.mol.intor('int1e_cg_irxp', comp=3, hermi=2)
+        )
```

Comment:
Can you format as in 217 to 218 ... I think in this case I would vote for brevity.

Reply:
also in the part below of course ;)

Reply:
I cannot because `line too long`.

Reply:
Cool ;)